external_search.py

```diff
@@ -52,95 +66,139 @@ def __init__(self, url=None, works_index=None):
             if not url:
                 raise Exception("Cannot connect to Elasticsearch cluster.")
 
-        self.works_index = self.__client.works_index
         self.indices = self.__client.indices
         self.search = self.__client.search
         self.index = self.__client.index
         self.delete = self.__client.delete
         self.exists = self.__client.exists
+
+        self.works_index = self.works_alias = self.__client.works_index
```

Comment:
I think it might make sense for the alias to be determined once, and set on the __client the same way the index is.